starter-code/src/component/FilterableProductTable.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import SearchBar from './SearchBar';
+import ProductTable from './ProductTable';
+
+export class FilterableProductTable extends Component {
+  render() {
```

Comment:
for the search: you did to create a function in the filterableProductTable component : this.props.product.filter(filter if the product name includes the search input value) and then you need to setState your currentList of products